docs/source/membership/membership.md

```diff
@@ -1,341 +1,197 @@
+
+## What role does an organization play in an MSP?
+
+An **organization** is a logical managed group of members. This can be something as big as a multinational corporation or a small as a flower shop. What's most important about organizations (or **orgs**) is that they manage their members under a single MSP. The MSP allows an identity to be linked to an organization. Note that this is different from the organization concept defined in an X.509 certificate, which we mentioned above.
+
+The exclusive relationship between an organization and its MSP makes it sensible to name the MSP after the organization, a convention you'll find adopted in most policy configurations. For example, organization `ORG1` would likely have an MSP called something like `ORG1-MSP`. In some cases an organization may require multiple membership groups --- for example, where channels are used to perform very different business functions between organizations. In these cases it makes sense to have multiple MSPs and name them accordingly, e.g., `ORG2-MSP-NATIONAL` and `ORG2-MSP-GOVERNMENT`, reflecting the different membership roots of trust within `ORG2` in the `NATIONAL` sales channel compared to the `GOVERNMENT` regulatory channel.
+
+### Organizational Units (OUs) and MSPs
+
+An organization can also be divided into multiple **organizational units**, each of which has a certain set of responsibilities. Think of an OU as a department
+inside an organization. For example, the `ORG1` organization might have both `ORG1.MANUFACTURING` and `ORG1.DISTRIBUTION` OUs to reflect these separate lines of business. When a CA issues X.509 certificates, the `OU` field in the certificate specifies the line of business to which the identity belongs. These values can then be used in policy definitions in order to restrict access or in smart contracts for attribute-based access control.
+
+Specifying OUs is optional. If OUs are not used, all of the identities that are part of an MSP --- as identified by the Root CA and Intermediate CA folders --- will be considered members of the organization.
+
+### Node OU Roles and MSPs
+
+Additionally, there is a special kind of OU, sometimes referred to as a `Node OU`, that can be used to confer a role onto an identity. These Node OU roles are defined in the `$FABRIC_CFG_PATH/msp/config.yaml` file and contain a list of organizational units whose members are considered to be part of the organization represented by this MSP. This is particularly useful when you want to restrict the members of an organization to the ones holding an identity (signed by one of MSP designated CAs) with a specific Node OU role in it. For example, an endorsement policy can specify that only members of ORG1 can endorse a transaction. But when Node OU roles are enabled, a more granular restriction can be implemented, requiring that only peers in ORG1 can endorse a transaction instead of any member of ORG1.
```

Comment:
I think that this sentance can be shortened for clarity: For example, an endorsement policy can specify that only members of ORG1 can endorse a transaction. But when Node OU roles are enabled, a more granular restriction can be implemented, requiring that only peers in ORG1 can endorse a transaction instead of any member of ORG1.

> For example, with node OU's you can implement a more granular endorsement policy to requires Org1 peers to endorse a transaction, rather than any member of Org1